R/file_utils.R

```diff
@@ -50,13 +50,14 @@ readlines_from_path <- function(pjnz_path, extension) {
   lines <- readLines(con)
 }
 
-#' Get data from DP file.
+#' Get csv data from file containing within PJNZ.
 #'
-#' @param pjnz_path Path to zip containing DP file.
+#' @param pjnz_path Path to PJNZ zip file.
+#' @param extension The file extension to look for.
```

Comment:
Tiny comment, but I would recommend giving an example here to remove ambiguity about whether `.` is required in the extension (`.DP` vs `DP`)